devel/README.md

```diff
@@ -33,10 +33,25 @@ cd image-builder-frontend/devel
 ./setup.sh
 ```
 
+## Environment Variables
+
```

Comment:
s/varaibles/variables/

Reply:
Euuh I got the line number wrong, I meant line 38: https://github.com/RedHatInsights/image-builder-frontend/pull/232/files#diff-7b200100e7a00a2d944f039757e38768138248b912d9d5603067ae272e1a27b1R38